plugins/auth/token.js

```diff
@@ -20,9 +20,7 @@ module.exports = () => ({
             scope: ['user', 'pipeline']
         },
         plugins: {
-            'hapi-swagger': {
-                security: [{ token: [] }]
-            }
+            'hapi-swagger': { security: [{ token: [] }] }
```

Comment:
Reduced number of lines so that there would no longer be a warning and thus the number of warnings would not exceed the maximum limit.

Is there something better that can be done instead?

Reply:
@InderH Please rebase.

Reply:
@adong do you want me to rebase specifically or is merging in master ok?

Reply:
I usually do rebase, but merging master is fine too.